### DISCUSSION TITLE
VueJs Split Button not working correctly

### DISCUSSION THREAD

**Discussion by Walnussbaer** [vuejs/language-tools#755] [Q&A] [answered]
Hello everyone, 

I'm using VSCode for a VueJs project. I have Volar installed. 

I want to use the split button provided by Volar which splits a Vue SCF into 3 editors, to be able to have a neat and structured working environment. 

But the split button only splits the SCF files into 2 editors: one with the template (HTML) and the other containing Style (SCSS) and the script (Typescript) 

Is this the intended behaviour? I would really like to have the split button split the SCF into 3 editors: CSS (SCSS), HTML and TS. Is this possible somehow?

**Comment by johnsoncodehk** [comment, marked as answer]
This behavior changed in [0.28.9](https://github.com/johnsoncodehk/volar/blob/master/CHANGELOG.md#0289), because split to 3 editors DX is not really good in my experience, split to 2 editors is more suitable for actual situation.

If you can downgrade to 0.28.8 to try split 3 editors if you interest in.

**Reply by mekefly** [reply]
ok ,Thank you

**Comment by Walnussbaer** [comment]
I guess this depends on the monitor resolution you are yousing. Could this be added as a config options, so that I can decide whether I want to use 2 or 3 editors?

**Reply by johnsoncodehk** [reply]
Could you try 0.28.8 for a while first? Because the difference is not only the number of splits, 0.28.9 use vscode split in group in single editor API, but this API do not support split to 3 editors. So it's split to multiple editors in 0.28.8.

**Reply by Walnussbaer** [reply]
all right, I understand what the problem here ist. I will try it out for a while. Thanks for the info! Really appreciate it :)

**Comment by crystalfp** [comment]
How about the test with 0.28.8? I'm currently at 0.35.2 and I'm not confident in a so big jump back just for testing.
Thanks!
mario

**Reply by johnsoncodehk** [reply]
@crystalfp Just give it a try and I'm think you will want come back to 2 editors version. 😅

**Reply by mattkenefick** [reply]
👎🏻
